content/en/continuous_integration/setup_pipelines/jenkins.md

```diff
@@ -145,9 +150,15 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
 5. Configure the `Global Job Tags`.
 6. Save your configuration.
 
-**Global tags** (`globalTags`): A comma-separated list of tags to apply to all metrics, traces, events, and service checks. Tags can include environment variables that are defined in the Jenkins controller instance: `global_tag:global_value,global_tag2:${SOME_ENVVAR},${OTHER_ENVVAR}:global_tagvalue`
+**Global tags**
+: A comma-separated list of tags to apply to all metrics, traces, events, and service checks. Tags can include environment variables that are defined in the Jenkins controller instance<br/>
+**Environment variable**: `DATADOG_JENKINS_PLUGIN_GLOBAL_TAGS`
+**Example**: `key1:value1,key2:${SOME_ENVVAR},${OTHER_ENVVAR}:value3`
 
-**Global job tags** (`globalJobTags`): A comma-separated list of regexes to match a job and a list of tags to apply to that job. Tags can include environment variables that are defined in the Jenkins controller instance. Tags can reference match groups in the regex using the `$` symbol: `(.*?)_job_(*?)_release:someValue`
+**Global job tags**
+: A comma-separated list of regexes to match a job and a list of tags to apply to that job. Tags can include environment variables that are defined in the Jenkins controller instance. Tags can reference match groups in the regex using the `$` symbol.
+**Environment variable**: `DATADOG_JENKINS_PLUGIN_GLOBAL_JOB_TAGS`
+**Example**: `(.*?)_job_(*?)_release, owner:$1, release_env:$2, optional:Tag3`
```

Comment:
I still think there's room for improvement. How would I go about having multiple regex values or is there only one? If there's only one, the text is a bit misleading.

Also, is `(*?)` valid regex syntax or should it be `(.*?)`? It might be good to specify what the regex syntax is specifically (Java probably? Or maybe it's PCRE...).

Anyway, this is a low-pri feedback.

Reply:
@drodriguezhdez do you know if it's possible to add more than one regex?

Reply:
> @drodriguezhdez do you know if it's possible to add more than one regex?
Yes, it's possible using `\r\n` as the separator. This is oriented to be configured using the Jenkins UI.